src/shared/mount-util.c

```diff
@@ -755,6 +755,7 @@ int bind_mount_in_namespace(
                 const char *dest,
                 int read_only,
                 int make_file_or_directory,
+                char **inaccessible_paths,
```

Comment:
hmm, why put this in this function? you can do this check trivially before hand no? and only one caller needs this...

Reply:
yeah not sure what I was thinking, moved out